tests/integration/test_log_proxy.py

```diff
@@ -51,3 +52,35 @@ async def test_alert_rules_do_fire_from_log_proxy(ops_test, loki_charm, log_prox
         for key in ["juju_application", "juju_model", "juju_model_uuid"]
         for alert in alerts
     )
+
+
+@pytest.mark.abort_on_fail
+async def test_check_both_containers_send_logs(ops_test, loki_charm, log_proxy_tester_charm):
```

Comment:
This is a consumer side PR, so we do not need to check for Loki's config.
We can check Promtail's config, but since we are sending logs to Loki we can say the config is OK.